backend/base/static/css/403.css

```diff
@@ -0,0 +1,41 @@
+@import url("https://fonts.googleapis.com/css?family=Bungee");
+
+body {
+  background: #1b1b1b;
+  color: white;
+  font-family: "Bungee", cursive;
+  margin-top: 50px;
+  text-align: center;
+}
+a {
+  color: #2aa7cc;
+  text-decoration: none;
+}
+a:hover {
+  color: white;
+}
+svg {
+  width: 50vw;
+}
+.lightblue {
+  fill: #444;
+}
+.eye {
+  cx: calc(115px + 30px * var(--mouse-x));
+  cy: calc(50px + 30px * var(--mouse-y));
+}
+#eye-wrap {
+  overflow: hidden;
+}
+.error-text {
+  font-size: 120px;
+}
+.alarm {
+  animation: alarmOn 0.5s infinite;
```

Comment:
what is alarmOn?

Reply:
Its for red light swirling effect on the top of 0.  A self defined function for adding effect via keyframes